onboardProject/atomic/atm/atm.list/list.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { StyledList, StyledListEntry } from './list.style';
+import { H2, Subline } from '../atm.typo/typo.style';
+
+export interface ListProps {
+    data: ListEntryKeyProps[];
+}
+
+export interface ListEntryKeyProps{
+    name: string;
+    email: string;
+}
+
+export interface ListEntryProps{
+    key: ListEntryKeyProps;
+}
+
+export const List = (props:ListProps) => 
+{
+    const renderListEntry = ({item}:ListEntryProps) => {
+        return <ListEntry email = {item.email} name = {item.name}/>
+    };
+
+    return (
+        <StyledList
+            data={props.data}
+            renderItem={renderListEntry}
+            keyExtractor = {(item:ListEntryKeyProps,index:number) => index.toString()}
```

Comment:
É bom evitarmos ao máximo usar index como key no React. Dá uma lida [nisso](https://reactjs.org/docs/lists-and-keys.html#keys). Como você acha que dá pra resolver?

Reply:
Assumindo que os "emails" dos usuários são únicos, indexar por esse valor?

Reply:
Creio que por e-mail funcione nesse caso. No entanto, geralmente a melhor alternativa é o `id`. Já te dando um spoiler: você precisará do `id` nessa listagem para uma tarefa futura, então já que tocamos no assunto, talvez seja uma boa você implementar.